config/types.ts

```diff
@@ -17,7 +17,7 @@ type WebConfig = {
     cors: Hapi.RouteOptionsCors
     security: Hapi.RouteOptionsSecureObject | boolean
   },
-  tls: null | { key: string, cert: string }
+  tls?: null | { key: string, cert: string }
```

Comment:
Haven't you removed this from the config?

Reply:
Yeah, we'll never use it. It's only useful if we deploy on a VPS or something else where there's no routing front-end that can handle things like SSL for us. We're on heroku now, and if we ever migrate it'll be to a similar cloud provider so I can't see it being needed.